portfolio/src/main/webapp/index.html

```diff
@@ -8,62 +8,131 @@
-      </p>
-      <h2>PROFILE</h2>
-      <p>
-          I am orginally from Banke district of Nepal. After completing my high school in 2019, I got 
-          admission from Howard University. I had to complete first year online while staying in home 
-          country due to Covid-19 and I arrived in USA in 2021. Currently, I am a rising junior pursuing 
-          my bachelor's degree in Computer Science at Howard University in Washington, D.C. I am an
-          enthusiastic Computer Science Student interested in Software develpoment, Machine Learning, 
-          and Artificial Intelligence.
-      </p>
-      <h2>PROJECTS</h2>
-        <h3>Online Shopping Application</h3>
-          <ul>
+    <nav class="navbar">
+        <ul>
+            <li><a href="#home">Home</a></li>
+            <li><a href="#profile">Profile</a></li>
+            <li><a href="#gallery">Gallery</a></li>
+            <li><a href="#projects">Projects</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+
+    <section id="home">
+        <div class="main">
+            <h1 class="headings">I AM <br> PREM RAJ OLI</h1>
+        </div>
+    </section>
+
+    <section id="profile">
+        <h1 class="headings">PROFILE</h1>
+        <div id="pic">
+            <img src="images/pp1.jpg">
+            <div id="intro">
+                <h2>PREM RAJ OLI</h2>
+                <p>
+                    I am orginally from Banke district of Nepal. After completing my high school in 2019, 
+                    I got admission from Howard University. I had to complete first year online while staying in 
+                    home country due to Covid-19 and I arrived in USA in 2021. Currently, I am a rising junior pursuing 
+                    my bachelor's degree in Computer Science at Howard University in Washington, D.C. I am an 
+                    enthusiastic Computer Science Student interested in Software develpoment, Machine Learning, 
+                    and Artificial Intelligence.
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section id="gallery">
+        <h1 class=headings>GALLERY</h1>
+        <div class="images">
+            <img src="images/g1.jpeg">
+            <img src="images/g2.jpeg">
+            <img src="images/g3.jpeg">
+            <img src="images/g4.jpeg">
+        </div>
+    </section>
+
+    <section id="projects">
+        <h1 class="headings">PROJECTS</h1>
+        <div class="project-div">
+            <h2>Online Shopping Application</h3>
+          <ul class="project-list">
             <li>
-                Built a database based shopping cart web application able to handle administrator 
-                and customer roles.
+                <p>
+                    Built a database based shopping cart web application able to handle administrator 
+                    and customer roles
+                </p>
             </li>
```

Comment:
I would fix the indentation here. It makes it hard to tell if the tags are matched.